release-team/role-handbooks/release-lead.md

```diff
@@ -34,6 +34,33 @@ A guiding principle for the lead is be an arbiter of decisions, and not the prim
 
 * Prior experience in release management is extremely helpful
 
+## Time Commitments:
+
+Release Lead is a very time-consuming role, especially towards the end of the release cycle.  Before you volunteer to be Release Lead, please make certain that your employer and your family are OK with you spending a lot of time on the release for the next three months.  Here's a rough estimate of the time requirements by week:
+
+* Weeks 1-4: 4-8 hours a week
+* Weeks 5-8: 6-12 hours a week
+* Weeks 9-13: 10 to 25 hours a week, including some after-hours and weekend work
+
+Among the specific time commitments you have are:
+
+* SIG-Release and Release Team meetings once a week during weeks 1-7.
+* Burndown meetings three to five times a week during weeks 8-12.
+* Community meetings once a week.
+
+## Choosing a Release Team
+
+One of your first, and definitely your most important, duties as Release Lead is to select a Release Team in consultation with SIG-Release.  Currently there are 10 roles to fill on the Release Team, plus one or more shadows (trainees) of each of those roles. each of which has specific requirements. Aside from any specifics, in general you are looking for candidates who:
+
+1. Can commit to the amount of time required across the release cycle;
+2. Are enthusiastic about being on the release team;
+3. Are members of the relevant SIGs for their role, if applicable;
+4. And have some prior experience with contributing to Kubernetes (such as shadowing a role for a prior release).
+
+In most cases, you will have a pool of candidates in the form of people who have previously shadowed the role in question, or have filled a different role on the Release Team and want to try something else.  Where possible, do select new team members over someone who has done the role several times, as we are trying to share experience across the organization.
+
+Also give some thought to ensuring diversity on the team, including members of more than one gender, ethnicity, nationality, and company affiliation.  This should not be hard to do, as prior release teams have had several women, several non-white folks, and staff from five or more companies.  In some cases, though, you may need to reach out to individuals you know and suggest that they apply for the team.
```

Comment:
I would use the term "underrepresented" in this context. Thank you for adding this.